machine/delta/delta_ag9064/machine.make

```diff
@@ -0,0 +1,46 @@
+# Delta AG9064 Series
+# CPU Module: Intel Atom Rangeley
+
+ONIE_ARCH ?= x86_64
+SWITCH_ASIC_VENDOR = bcm
+
+VENDOR_REV ?= 0
+
+# Translate hardware revision to ONIE hardware revision
+ifeq ($(VENDOR_REV),0)
+  MACHINE_REV = 0
+else
+  $(warning Unknown VENDOR_REV '$(VENDOR_REV)' for MACHINE '$(MACHINE)')
+  $(error Unknown VENDOR_REV)
+endif
+
+# Vendor ID -- IANA Private Enterprise Number:
+# http://www.iana.org/assignments/enterprise-numbers
+# AGEMA IANA number
+VENDOR_ID = 5324
+
+I2CTOOLS_ENABLE = yes
+#I2CTOOLS_SYSEEPROM = no
+I2CTOOLS_SYSEEPROM = yes
+
+# This platform requires the PXE_EFI64 installer
+PXE_EFI64_ENABLE = yes
+
+CONSOLE_SPEED = 115200
+CONSOLE_DEV = 0
+CONSOLE_FLAG = 0
```

Comment:
`CONSOLE_FLAG` is not used.  Please remove.

Reply:
OK